cmd/osm-controller/validate_test.go

```diff
@@ -1,147 +0,0 @@
-package main
```

Comment:
if this test file is being deleted shouldn't the `validate.go` file also be deleted in this PR ?

Reply:
Let me add back validate_test.go. osm-injector is still using methods in validate.go